frontend/src/layout/onboarding/onboarding.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Popover, Button, Checkbox, Badge, Modal } from 'antd'
+import { useValues, useActions } from 'kea'
+import { actionsModel } from '~/models/actionsModel'
+import { Loading } from 'lib/utils'
+import { StarOutlined, StarFilled } from '@ant-design/icons'
+import { Link } from 'lib/components/Link'
+import FunnelImage from '../_assets/funnel_with_text.png'
+import ActionImage from '../_assets/actions.png'
+import TrendImage from '../_assets/trend.png'
+import { onboardingLogic, TourType } from './onboardingLogic'
+import { userLogic } from 'scenes/userLogic'
+import _ from 'lodash'
+import api from 'lib/api'
+
+export function OnboardingWidget() {
+    const contentRef = useRef()
+    const { actions, actionsLoading } = useValues(actionsModel)
+    const [instructionalModal, setInstructionalModal] = useState(false)
+    const { user } = useValues(userLogic)
+    const { loadUser } = useActions(userLogic)
+    const { tourType, checked } = useValues(onboardingLogic({ user }))
+    const { setTourActive, setTourType, updateOnboardingInitial } = useActions(onboardingLogic)
+    const [visible, setVisible] = useState(user.onboarding.initial ? true : false)
+
+    const unfinishedCount = _.filter(checked, (isChecked) => !isChecked).length
+
+    function closePopup() {
+        if (user.onboarding.initial) updateOnboardingInitial(false)
+        setVisible(false)
+    }
+
+    let onClickOutside = (event) => {
+        if (contentRef.current && !contentRef.current.contains(event.target)) {
+            closePopup()
+        }
+    }
+
+    useEffect(() => {
+        document.addEventListener('mousedown', onClickOutside)
+        return () => {
+            document.removeEventListener('mousedown', onClickOutside)
+        }
+    }, [])
+
+    async function dontShowAgain() {
+        await api.update('api/user', { onboarding: { ...user.onboarding, active: false } })
+        loadUser()
+    }
+
+    function content() {
+        return (
+            <div
+                data-attr="onboarding-content"
+                ref={contentRef}
+                style={{ display: 'flex', width: '25vw', flexDirection: 'column' }}
+            >
+                <h2>Get Started</h2>
+                <i>
+                    Complete these steps to learn how to use Posthog! Click on an item below to learn how to create one
+                </i>
+                {Object.entries(TourType).map(([, value], index) => {
```

Comment:
`value` is a pretty good name because we're in a small scope, but I wonder if `tourType` would be marginally better.